### DISCUSSION TITLE
Intellisense Code Completion

### DISCUSSION THREAD

**Discussion by Btibert3** [posit-dev/positron#5002] [Q&A] [answered]
I am using Positron for a python programming class I am running this semester, and I have had a few students ask me about why Intellisense is not firing on their system.  For one student (on Windows), I confirmed that the file paths were not appearing when trying to use pd.read_csv("").  The file was indeed in the working directory and could load the file; Intellisense just didn't fire for them.

Are there settings or configurations that may need to be reviewed?  In my case, I am on a Macbook and this works without issue.

**Comment by jmcphers** [comment, marked as answer]
Positron's path completion on Windows is powered by Jedi, which has some known issues. Was your student in a Jupyter notebook? See this issue for details: https://github.com/posit-dev/positron/issues/3758

**Comment by Btibert3** [comment]
Yes, the issue is similar.  And as luck would have it, on another student's computer I saw what is described here: https://github.com/posit-dev/positron/issues/3758#issuecomment-2246733260

And yes, they were using a Jupyter notebook via New > Jupyter Notebook from within Positron.

I will close this as it's related to an open issue.
